dtests/src/test/scala/io/snappydata/hydra/SnappyTestUtils.scala

```diff
@@ -85,8 +95,19 @@ object SnappyTestUtils {
   }
 
   def assertQueryFullResultSet(snc: SnappyContext, sqlString: String, queryNum: String,
-                               tableType: String, pw: PrintWriter, sqlContext: SQLContext): Any = {
-    var snappyDF = snc.sql(sqlString)
+      tableType: String, pw: PrintWriter, sqlContext: SQLContext): Any = {
+    assertQueryFullResultSet(snc, sqlString, queryNum, tableType, pw, sqlContext, true)
+  }
+
+  def assertQueryFullResultSet(snc: SnappyContext, sqlString: String, queryNum: String,
+      tableType: String, pw: PrintWriter, sqlContext: SQLContext,
+      usePlanCaching: Boolean): Any = {
+    var snappyDF: DataFrame = null
+    if(!usePlanCaching) {
+      snappyDF = snc.sqlUncached(sqlString)
+    } else {
+      snappyDF = snc.sql(sqlString)
+    }
     var sparkDF = sqlContext.sql(sqlString);
```

Comment:
Add common code in a separate method and use the same in both the assertQueryFullResultSet methods.

Reply:
The code is added in the common method, only for signature, a new method is added.